CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ### Added
 
+- Define a general Resources Detector Interface and allow resources to be defected from environment variables under folder `sdk/resource`. (#939)
```

Comment:
s/defected/tetected/. the subpath doesn't matter for changelog, people can read that out of the PR description; paths mentioned in pulls for where the location _is_ significant (e.g. a previous API being removed).

Reply:
Going to resolve and we can finalize wording here during the release.